internal/services/formatters/interface.go

```diff
@@ -82,7 +83,7 @@ type IService interface {
 	AddNewVulnerabilityIntoAnalysis(vulnerability *vulnerability.Vulnerability)
 
 	// GetCustomRulesByLanguage return user custom rules to a given language.
-	GetCustomRulesByLanguage(lang languages.Language) []engine.Rule
+	GetCustomRulesByLanguage(lang languages.Language) []text.Rule
```

Comment:
Considering the idea above this should method should continue returning `[]engine.Rule`